packages/react/src/button/button.tsx

```diff
@@ -1,33 +1,28 @@
-import { forwardRef } from "react";
+import { forwardRef, Children } from "react";
 import { clsx } from "@postenbring/hedwig-css/typed-classname";
 import { Slot } from "@radix-ui/react-slot";
 
 export interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   /**
    * The height, font size and padding of the button
+   *
+   * @default "large"
    */
-  size?: "small" | "medium" | "large";
+  size?: "small" | "large";
 
   /**
    * The background and fill of the button
    *
    * @default "primary"
    */
-  variant?: "primary" | "secondary" | "primary-outline" | "secondary-outline";
+  variant?: "primary" | "secondary" | "tertiary" | "inverted";
```

Comment:
💥 Breaking change

<img width="1507" alt="image" src="https://github.com/user-attachments/assets/1c719126-8068-414c-984a-25b0b727968f" />

We either have to decide to do a major version (along with link) or do the gradual change with a deprecation warning
see this thread https://posten.slack.com/archives/G01AD31F1QC/p1740744085172999

Reply:
I've deprecated the old variants